### PR TITLE
Fix mismatch extractorname

### PR DESCRIPTION
**Pull request by bingzhang**
## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the CHANGELOG.md.
- [ ] I have signed the CLA
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by lmarini — CHANGES_REQUESTED** [review]
I believe this should be against the 1.12.0 branch. https://github.com/clowder-framework/clowder/pull/108

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Fixed
+- fix the mismatch extractor name and extractor service name in log query.
```
Comment (robkooper) [inline CHANGELOG.md:10]: no need to add a fix, since this was never released.

**Comment by robkooper** [comment]
Do you have a docker-compose.logging.yml you can add so it is easy to test for others?

**Comment by bingzhang** [comment]
> Do you have a docker-compose.logging.yml you can add so it is easy to test for others? (non swarm please).

I have local test

> I believe this should be against the 1.12.0 branch. #108

Yes, someone already changed the destination to release/1.12.0.

**Comment by bingzhang** [comment]
> > I believe this should be against the 1.12.0 branch. #108
> 
> Yes, someone already changed the destination to release/1.12.0.

**Comment by bingzhang** [comment]
close.

**Pull request closed without merging**